Sourcery/Models/Method.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+class AMethod: NSObject {
```

Comment:
I'm not a fan of `AMethod` name, but using `Method` causes ambiguities. Can you suggest any better name?